tests/ampctl_parse.c

```diff
@@ -1393,7 +1393,7 @@ int ampctl_parse(AMP *my_amp, FILE *fin, FILE *fout, char *argv[], int argc)
     pthread_mutex_unlock(&amp_mutex);
 #endif
 
-    if (retcode == RIG_EIO) { return retcode; }
```

Comment:
The new check is ok, but the return is a different bug (it should return `2` meaning Hamlib error, so I'm going to open a new PR).